### PR TITLE
NXP-20773 & NXP-22588: FT for user cloud services and authorized apps

### PR DESCRIPTION
**Pull request by Gabez0r**
Depends on:
- nuxeo/nuxeo#1888
- nuxeo/nuxeo-web-ui#186

### CODE REVIEW

File: nuxeo-web-ui-ftest/wdio.conf.js [hunk 1]
```diff
@@ -141,7 +143,7 @@ exports.config = {
     profile: [],        // <string[]> (name) specify the profile to use
     strict: true,      // <boolean> fail if there are any undefined or pending steps
     tags: [],           // <string[]> (expression) only execute the features or scenarios with tags matching the expression
-    timeout: 20000,     // <number> timeout for step definitions
+    timeout: process.env.DEBUG ? 24 * 60 * 60 * 1000 : 20000,     // <number> timeout for step definitions
```
Comment (guirenard) [inline nuxeo-web-ui-ftest/wdio.conf.js]: 24h timeout for a single step? :scream: What's the point?

**Comment by guirenard** [comment]
:+1:

File: nuxeo-web-ui-ftest/test/pages/ui/oauth2/user_authorized_apps.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+import BasePage from '../../base';
+
+class App {
```
Comment (afjusto) [inline nuxeo-web-ui-ftest/test/pages/ui/oauth2/user_authorized_apps.js]: I don't like the name, put something more specific if possible

**Comment by afjusto** [comment]
👍

File: nuxeo-web-ui-ftest/wdio.conf.js (hunk 1 above)
Comment (nelsonsilva) [reply nuxeo-web-ui-ftest/wdio.conf.js]: I guess it's to allow for you to set breakpoints and debug...

File: nuxeo-web-ui-ftest/test/pages/ui/oauth2/user_authorized_apps.js (hunk 2 above)
Comment (Gabez0r) [reply nuxeo-web-ui-ftest/test/pages/ui/oauth2/user_authorized_apps.js]: I can call it AuthorizedApp, but keep it mind that this class won't be exported.

**Comment by afjusto** [comment]
👍

**Pull request merged**